draft-ietf-scitt-architecture.md

```diff
@@ -397,7 +397,7 @@ Issuer:
 
 : an identifier representing an organization, device, user, or entity securing Statements about supply chain Artifacts.
 An Issuer may be the owner or author of Artifacts, or an independent third party such as an Auditor, reviewer or an endorser.
-In SCITT Statements and Receipts, the `iss` CWT Claim is a member of the COSE header parameter `15: CWT Claims` as defined in {{-CWT_CLAIMS_COSE}} within the protected header of a COSE Envelope.
+In SCITT Statements and Receipts, the `iss` CWT (CBOR Web Token) Claim is a member of the COSE header parameter `15: CWT Claims` as defined in {{-CWT_CLAIMS_COSE}} within the protected header of a COSE Envelope.
```

Comment:
this is a bit awk though, as "CWT Claims Set" is the "term"... 

https://datatracker.ietf.org/doc/html/rfc8392

Reply:
How about #449?